src/modules/esl-base-element/core/esl-base-element.ts

```diff
@@ -108,16 +108,16 @@ export abstract class ESLBaseElement extends HTMLElement implements ESLBaseCompo
    */
   public static register(this: typeof ESLBaseElement, tagName?: string): void {
     tagName = tagName || this.is;
-    if (!tagName) throw new Error('Can not define custom element');
-    const constructor: any = customElements.get(tagName);
-    if (constructor) {
-      if (constructor.is !== tagName) throw new Error('Element declaration tag inconsistency');
-      return;
+    if (!tagName) {
+      throw new DOMException('[ESL]: Incorrect tag name', 'NotSupportedError');
```

Comment:
Maybe a should be implemented message formatting method for consistency purposes?

Reply:
Agree I will create a technical cleanup regarding the unification of exception handling